setup.py

```diff
@@ -1149,6 +1149,8 @@ def main():
         "jinja2",
         "fsspec",
     ]
+    if IS_WINDOWS:
+        install_requires.append("mkl==2021.4.0")
```

Comment:
Is an an exact pin needed here? I don't see why can't it be 2021.4.0 or later?

Reply:
MKL has the version in its name. Since we still link with the static version of the dlls it will search for dlls with 1 in their name. So if we install MKL 2023.1.0 it will have those libraries but we 2 in their names and it will fail to load. So based on my tests the only thing compatible MKL versions are 2021.1.1-2021.4.0.

Reply:
Thanks, I figured you had a good reason here.